### PR TITLE
Plasteel Combat Helmet - Shitty Civvie Hats, Because Why Not

### PR DESCRIPTION
**Pull request by Addust**
## About The Pull Request

Adds the stamped plasteel combat helmet, a cheap (2 plasteel) helmet that is not good at staying on your head (can't fall off on its own, but can be stripped nearly instantly), and, if combined with leather, a lined stamped plasteel combat helmet, which is basically that but with lengthened strip time and slightly increased armour. Needs a welder for the unlined version.

i need spriting advice

front view, unlined is the same without the straps
image

## Why It's Good For The Game

Ghetto helmets, gentlemen? Gives people unique solutions for headgear while on planets.

## Changelog

:cl:
add: A new craftable helmet has been added! Costs 2 plasteel, you can also craft a lined version with some leather.
/:cl:

### CODE REVIEW

**Comment by BarteG44** [comment]
i applaud the soul

**Review by rye-rice — CHANGES_REQUESTED** [review]
Repeating what I said on discord:

To be honest,  if you're going to make a makeshift helmet, make it look makeshift instead of making it look like a ww1 helmet.

The Eris makeshift helmet should give some ideas

The sprite's is a bit flat but I'd just make up a new design instead of touching it up 
For the obj, check how the other helmets are drawn, 3/4 to the side


image

File: _maps/outpost/outpost_test_2.dmm [hunk 1]
```diff
@@ -261,6 +261,10 @@
 /obj/structure/statue/snow/snowman{
 	pixel_y = 5
 	},
+/obj/item/clothing/head/helmet/stamped_steel/lined{
+	pixel_y = 13;
+	pixel_x = -1
+	},
```
Comment (goober3) [inline _maps/outpost/outpost_test_2.dmm]: do not add a helmet to the outpost. thanks

**Review by goober3 — CHANGES_REQUESTED** [review]

**Comment by Bimbly** [comment]
Plasteel is nigh impossible to get without an ORM which I feel kinda goes against the spirit of it being makeshift thrown together. I personally would have it made out of normal metal like the scrap armor. (also strip time isnt much of a downside if you want it to have a real one maybe have it make you nearsighted or something?)

**Comment by thgvr** [comment]
Not a huge fan of the sprite, or the reliance on crafting menu stuff. I am also not convinced this has a place.

**Comment by Imaginos16** [comment]
Not particularly impressed by the PR, and Mark shared much of the same sentiments as I and the other reviewers. As such, I'll be closing the PR.

**Pull request closed without merging**